Library/Homebrew/services/formula_wrapper.rb

```diff
@@ -234,7 +234,7 @@ def load_service
 
       def status_output_success_type
         @status_output_success_type ||= if System.launchctl?
-          cmd = [System.launchctl.to_s, "list", service_name]
+          cmd = [System.launchctl.to_s, "list", "#{System.domain_target}/#{service_name}"]
```

Comment:
`launchctl list` doesn't accept this syntax AFAICT, so this basically means we're always `launchctl print` now.

Which is probably better, though does come with the caveat:

> IMPORTANT: This output is NOT API in any sense at all. Do NOT
              rely on the structure or information emitted for ANY reason. It
              may change from release to release without warning.

But Apple haven't exactly provided any alternative API...

Reply:
> Do we eliminate `launchctl list` entirely?

Could do. Alternatively: we revert this line and look at both only if we cannot get a PID/successful run state from `list`

The issue this was working around was one in which `launchctl list` said a service was not running which `launchctl print` (correctly) said was running

Reply:
> `launchctl print` is probably better, though does come with the caveat:

`launchctl list` comes with the caveat:

> Recommended alternative subcommand: print

so I'm tempted to say we should follow this direction of travel

Reply:
I've switched the ordering in https://github.com/Homebrew/brew/pull/19692